sources/targets/Stride.Core.props

```diff
@@ -15,15 +15,13 @@
 
     <!-- Default values -->
     <StridePlatformOriginal>$(StridePlatform)</StridePlatformOriginal>
-    <StridePlatform Condition=" $(TargetFramework.StartsWith('net')) ">Windows</StridePlatform>
-    <StridePlatform Condition=" $(TargetFramework.StartsWith('net')) And $(RuntimeIdentifier.StartsWith('linux')) ">Linux</StridePlatform>
-    <StridePlatform Condition=" $(TargetFramework.StartsWith('net')) And $(RuntimeIdentifier.StartsWith('osx')) ">macOS</StridePlatform>
+    <StridePlatform>Windows</StridePlatform>
+    <StridePlatform Condition=" '$([MSBuild]::IsOSPlatform(Linux))' ">Linux</StridePlatform>
```

Comment:
Wouldn't that prevent building for Linux from a Windows machine?

Reply:
@Kryptos-FR Huh...Yes, Good point. I realized that Linux detection gonna work only on Linux but on Windows not really

Reply:
I bringed back RuntimeIdentifier then